ui/app/mainui/AppMain.qml

```diff
@@ -899,6 +894,34 @@ Item {
         }
     }
 
+    ListView {
+        id: toastArea
+        anchors.top: parent.top
```

Comment:
It depends QML code-style and personal taste, to use here [grouped properties](https://het.as.utexas.edu/HET/Software/html/propertybinding.html#qml-grouped-properties):
`
anchors {  
top: parent.top; 
topMargin: 60;  
right: parent.right;  
rightMargin: 8;  
}
`

Reply:
> Maybee re-assign review on someone else?

@MishkaRogachev there are more reviewers, don't worry, thanks. ;)

> It depends QML code-style and personal taste, to use here [grouped properties](https://het.as.utexas.edu/HET/Software/html/propertybinding.html#qml-grouped-properties):

As you said, depends on personal preferences, I use grouped properties in some specific cases only when I see it readable, but most of the time I don't group them.